src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -82,6 +82,9 @@ export class EditDataGridPanel extends GridParentComponent {
 	// Prevent the cell submission function from being called multiple times.
 	private cellSubmitInProgress: boolean;
 
+	// Manually submit the cell after edit end if it's the null row.
+	private isLastRowEnter: boolean;
```

Comment:
Consider naming this `isInNullRowEnter` 

1. `isIn` is usually used to indicate state (which this is)
2. We call it the null row everywhere else so good to keep it consistent